app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    input_command, file_in, file_out = command.split()
```

Comment:
If the command will be for example `cp file1.txt file2.txt file3.txt.`
I recommend you to validate command